l10n/en/firefox/browsers.ftl

```diff
@@ -41,7 +41,7 @@ firefox-browsers-join-firefox-and-get-the-most = Join { -brand-name-firefox } an
 # Variables:
 #   $fxa_attr (string) - anchor link url and attributes
 #   $accounts_attr (string) - anchor link url and attributes
-firefox-browsers-already-have-an-account-sign = Already have an account? <a { $fxa_attr }>Sign In</a> or <a { $accounts_attr }>learn more</a> about joining Firefox.
+firefox-browsers-already-have-an-account-sign = Already have an account? <a { $fxa_attr }>Sign In</a> or <a { $accounts_attr }>learn more</a> about joining { -brand-name-firefox }.
```

Comment:
Note for reviewer: I'm resisting using new string IDs for these changes as we're only fixing a few missing brand name placeholders, rather than changing the sentences themselves. But I'd be happy to use new strings instead if we think that's worthwhile to notify translators.

Reply:
You need to use a new ID, because a missing placeable is a warning in Pontoon, and all existing translations will get one because of these changes.